recipes/smokesignal/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "smokesignal" %}
+{% set version = "0.7.0" %}
+{% set sha256 = "8e0e651a73c258106e59bf8574090fc1cf8a903ee2d074616b76c3a170d53396" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```

Comment:
Could this be a noarch pacakge?

Reply:
My org is having issues including noarch packages that is complicated to describe here. Thus, if I build it noarch we won't be able to use the package on our side.

I hope it is still okay to build it for each architecture separately.

Reply:
Its fine, we'll just have to wait for Travis then.

Reply:
Out of curiosity what were the issues you were having with noarch?

Reply:
Hi,
It looks like my build failed for no reasons related to my package. I will skip the OSX build for now, and add it back when the package is fully integrated.